docs/recipe-usage.md

```diff
@@ -0,0 +1,195 @@
+# Using Recipes
+
+Megatron Bridge provides production-ready training recipes for several popular models. You can find an overview of supported recipes and 🤗 HuggingFace bridges [here](index.md#supported-models).
+This guide will cover the next steps to make use of a training recipe, including how to [override configuration](#overriding-configuration) and how to [launch a job](#launch-methods).
+
```

Comment:
this doc talks a lot about configs, but not really about the recipes themselves. a description of how we provide recipes would be useful. for instance, we provide training recipes for each model family/size, which includes a pretraining config (and soon a finetuning recipe config). 

these recipes are designed to work with the configs here: https://docs.nvidia.com/nemo/megatron-bridge/latest/training/entry-points.html

and that these recipes represent our best known defaults for convergence and performance